deploy/olm-catalog/service-assurance-operator/0.1.0/infra.watch_serviceassurances_crd.yaml

```diff
@@ -20,62 +20,18 @@ spec:
-          type: string
-        kind:
-          description: 'Kind is a string value representing the REST resource this
-            object represents. Servers may infer this from the endpoint the client
-            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
-          type: string
-        metadata:
-          description: Metadata definition for the ServiceAssurance object
-          type: object
         spec:
           description: Specification of the desired behavior of the Service Assurance Operator.
           properties:
-            metricsEnabled:
-              description: Whether the Service Assurance Operator should enable components related to metrics collection and storage.
-              type: boolean
-            eventsEnabled:
-              description: Whether the Service Assurance Operator should enable components related to events collection and storage.
-              type: boolean
-            haEnabled:
-              description: Whether to deploy the services in HA mode.
-              type: boolean
-            esNodeCount:
-              description: Number of elastic search instances to deploy in HA mode (ignored unless haEnabled == True)
-              type: string
-            prometheusStorageClass:
-              description: Storage class name used for Prometheus PVC
-              type: string
-            prometheusStorageResources:
-              description: Storage resource definition for Prometheus
-              type: string
-            prometheusStorageSelector:
-              description: Storage selector definition for Prometheus
-              type: string
-            prometheusPvcStorageRequest:
-              description: PVC storage requested size for Prometheus
-        status:
-          description: Status results of an instance of Service Assurance
-          properties:
-            conditions:
-              description: The resulting conditions when a Service Assurance is instantiated
-              items:
-                properties:
-                  status:
-                    type: string
-                  type:
-                    type: string
-                  reason:
-                    type: string
-                  lastTransitionTime:
-                    type: string
-                type: object
-              type: array
-          type: object
-
-# vim: set tabstop=2 shiftwidth=2 expandtab ft=yaml:
+            metrics:
+              description: Options related to metrics collection and storage.
+              properties:
+                enabled:
+                  description: Whether the Service Assurance Operator should enable components related to metrics collection and storage.
+                  type: boolean
+            events:
```

Comment:
Why is this showing up as a nested value? Everything needs to be flat with the Ansible operator. The configuration you're referencing in the CR is `eventsEnabled` and `metricsEnabled`, so that's what needs to exist here. I'm going to call it out further down as well.